### PR TITLE
First optimism decoder -- hop eth withdrawal

### PR DESCRIPTION
**Pull request by LefterisJP**
Also quite a few more things

### CODE REVIEW

File: rotkehlchen/db/minimized_schema.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
...
     'evm_transactions': 'tx_hashBLOBNOTNULL,chain_idINTEGERNOTNULL,timestampINTEGERNOTNULL,block_numberINTEGERNOTNULL,from_addressTEXTNOTNULL,to_addressTEXT,valueTEXTNOTNULL,gasTEXTNOTNULL,gas_priceTEXTNOTNULL,gas_usedTEXTNOTNULL,input_dataBLOBNOTNULL,nonceINTEGERNOTNULL,PRIMARYKEY(tx_hash,chain_id)',
-    'evm_internal_transactions': 'parent_tx_hashBLOBNOTNULL,chain_idINTEGERNOTNULL,trace_idINTEGERNOTNULL,timestampINTEGERNOTNULL,block_numberINTEGERNOTNULL,from_addressTEXTNOTNULL,to_addressTEXT,valueTEXTNOTNULL,FOREIGNKEY(parent_tx_hash,chain_id)REFERENCESevm_transactions(tx_hash,chain_id)ONDELETECASCADEONUPDATECASCADE,PRIMARYKEY(parent_tx_hash,chain_id,trace_id)',
+    'evm_internal_transactions': 'parent_tx_hashBLOBNOTNULL,chain_idINTEGERNOTNULL,trace_idINTEGERNOTNULL,timestampINTEGERNOTNULL,block_numberINTEGERNOTNULL,from_addressTEXTNOTNULL,to_addressTEXT,valueTEXTNOTNULL,FOREIGNKEY(parent_tx_hash,chain_id)REFERENCESevm_transactions(tx_hash,chain_id)ONDELETECASCADEONUPDATECASCADE,PRIMARYKEY(parent_tx_hash,chain_id,trace_id,from_address,to_address,value)',
```
Comment (yabirgb) [inline rotkehlchen/db/minimized_schema.py:20]: Shouldn't this also go in the packaged DB? we need to remember it. I'll add a note

Comment (LefterisJP) [reply rotkehlchen/db/minimized_schema.py:20]: Why does this need to go in the packaged DB? I don't understand what you are talknig about

Comment (LefterisJP) [reply rotkehlchen/db/minimized_schema.py:20]: The failed test probably shows me you meant to write about hETH? That's a different file ...

**Comment by LefterisJP** [comment]
All is green. This introduced a few changes which I describe both in the commits and also in discord. Merging.

**Pull request merged**